packages/embark-ens/src/index.js

```diff
@@ -373,16 +375,15 @@ class ENS {
       }
 
       this.events.request('code-generator:ready', () => {
-        this.events.request('code-generator:symlink:generate', location, 'eth-ens-namehash', (err, symlinkDest) => {
+        this.events.request('code-generator:symlink:generate', location, 'eth-ens-namehash', (err) => {
           if (err) {
             this.logger.error(__('Error creating a symlink to eth-ens-namehash'));
             return this.logger.error(err.message || err);
           }
           this.events.emit('runcode:register', 'namehash', require('eth-ens-namehash'), () => {
-            let code = `\nconst namehash = global.namehash || require('${symlinkDest}');`;
-            code += this.fs.readFileSync(joinPath(__dirname, 'ENSFunctions.js')).toString();
-            code += "\n" + this.fs.readFileSync(joinPath(__dirname, 'embarkjs.js')).toString();
-            code += "\nEmbarkJS.Names.registerProvider('ens', __embarkENS);";
+            let code = "";
+            code += "\nconst __embarkENS = require('embarkjs-ens')";
```

Comment:
I guess this requires the Dapp to install `embarkjs-ens` like whisper and ipfs. 
Again, if it breaks, like I expect it does, we should probably do something about it. Like wrapping it in a try/catch.